public/js/todoController.js

```diff
@@ -0,0 +1,17 @@
+var todoListApp = angular.module('todoList', []);
+
+todoListApp.controller('todoController', function($scope){
+
+  $scope.allTasks = [];
+
+  $scope.add = function(){
+  
+    $scope.allTasks.push({
+      text: $scope.newTask,
+      done: false
+    });
+  
+  $scope.newTask = '';
```

Comment:
Lines 14 and 15 should be indented one tab further because they're inside the add function